app/components/thumbnail.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+const MAX_THUMBNAIL_DIMENSION = 999;
+
+export default class Thumbnail extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      failed: false,
+    };
+  }
+
+  getThumbnailSrc({ origin, width, height, src }) {
+    const srcPath = src.split('//').pop();
+    return (`${origin}/${width}x${height}/${srcPath}`);
+  }
+
+  handleError() {
+    if (!this.state.failed) {
```

Comment:
when attempting to upload the FilterIcons (.svg's) from Ali's sample survey task via Project Builder, Media or Survey Task editor, console had 415 errors and the following error message:
```
thumbnail.jsx?bf6a:20 Uncaught TypeError: Cannot read property 'state' of null(…)
```

Uploading ReferencePhotos (.jpg's) worked great.